module/os/linux/zfs/zfs_ioctl_os.c

```diff
@@ -312,6 +312,11 @@ MODULE_ALIAS("zunicode");
 MODULE_ALIAS("zcommon");
 MODULE_ALIAS("zzstd");
 MODULE_DESCRIPTION("ZFS");
+MODULE_AUTHOR("Lua: Lua.org, PUC-Rio");
+MODULE_AUTHOR("zstd: Facebook, Inc.");
```

Comment:
As standalone modules I think this made sense, but with everything now merged together it seems a bit odd to call out just these two additional authors.  In reality there are many more authors for important components (i.e. lz4, zlib, aes, skein, cityhash, etc).  I'd suggest we drop these and rely on the more complete attribution in the source code.

Reply:
Should the additional licences stay, or should I get rid of them as well?

Reply:
Let's keep the licenses.

Reply:
Rebased, authors removed